src/components/AnalysisPage/UploadPartButton/UploadPartButton.jsx

```diff
@@ -33,7 +29,7 @@ const UploadPartButton = ({
       if (status === "done") {
         message.success(`${info.file.name} file uploaded successfully.`);
         setPartId(info.file.response.id);
-        setPartsThresholdHistory([]);
+        setThickData([]);
```

Comment:
TODO: Is setting the thickness data back to an empty array necessary in the UploadPartButton? Why doesnt the geom analysis data have to be set back to null here?

Reply:
No it was not necessary, 

It was definitely an oversight by me, just vibing last night. 

I should have set it back to an empty array in the useEffect hook with partId as a dependency in ThickAnalysisBox.jsx. It has the same effect but less passing of states.

I fixed it!

Reply:
Ok cool good call :+1: 
Where did you fix it?

Reply:
I removed setListOfThicknessData([]); (originally setThickData([])) from 
![image](https://github.com/rknizzle/propermesh-view/assets/111093132/2ea646dd-7417-466d-a325-dfb80b4988b2)
 in UploadPartButton.jsx. also removed the passing of the prop.

and just called it in the useEffect hook in ThickanalysisBox.jsx that has partId has a dependency:
![image](https://github.com/rknizzle/propermesh-view/assets/111093132/7cc2e55c-39aa-45af-85c7-d91b4db52b7a)

this is exactly how geoBox is resetting geoData. 

I think I was in some sort of zone last night. Making myself do extra work hahah

Reply:
Oh ok sweet I just couldn't find that change in the code. Did you push that commit yet?

Reply:
Ok there it is I see the commit now! This is all good